DEVELOPMENT.rst

```diff
@@ -0,0 +1,39 @@
+Developing on edx-sga
+==============================
+
+Setup (including devstack setup)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Install vagrant: http://docs.vagrantup.com/v2/installation/
+2. Install virtualbox: https://www.virtualbox.org/wiki/Downloads
+3. ``mkdir devstack``
+4. ``cd devstack``
+5. ``curl -L https://raw.githubusercontent.com/edx/configuration/master/vagrant/release/devstack/Vagrantfile > Vagrantfile``
+6. ``vagrant plugin install vagrant-vbguest``
+7. ``vagrant up``
+8. ``cd themes/``
+9. Fork https://github.com/mitodl/edx-sga.git to your own github account.
+10. ``git clone https://github.com/your-name/edx-sga.git``
+11. ``vagrant ssh``
+12. ``sudo su edxapp``
+13. ``cd /edx/app/edxapp/themes/``
+14. ``pip uninstall edx-sga`` (since it's part of the edx distribution, we have to remove the installed version)
+15. ``cd edx-sga/``
+16. ``pip install -e .``
+17. ``paver run_all_servers``
+
+You should now see your fork of the most recent master branch of edx-sga running in the LMS.
+
+Developing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
nit: strict .RST rules require that the formatting line is the same length as the text it formats.